templates/decision/record.tpl

```diff
@@ -52,89 +52,81 @@
-						attach-files-label="{translate key="common.attachFiles"}"
-						attached-files-label="{translate key="semicolon" label=$attachedFilesLabel}"
-						:attachers="step.attachers"
-						:attachments="step.attachments"
-						:bcc="step.bcc"
-						bcc-label="{translate key="semicolon" label=$bccLabel}"
-						:body="step.body"
-						body-label="{translate key="stageParticipants.notify.message"}"
-						:can-change-recipients="step.canChangeRecipients"
-						:cc="step.cc"
-						cc-label="{translate key="semicolon" label=$ccLabel}"
-						confirm-switch-locale-label="{translate key="email.confirmSwitchLocale"}"
-						deselect-label="{translate key="common.deselect"}"
-						:email-templates="step.emailTemplates"
-						:email-templates-api-url="emailTemplatesApiUrl"
-						find-template-label="{translate key="common.findTemplate"}"
-						:id="step.id"
-						:initial-template-key="step.initialTemplateKey"
-						insert-label="{translate key="common.insert"}"
-						insert-modal-label="{translate key="common.insertContent"}"
-						insert-content-label="{translate key="common.content"}"
-						insert-search-label="{translate key="common.insertContentSearch"}"
-						load-template-label="{translate key="common.loadTemplate"}"
-						:locale="step.locale"
-						:locales="step.locales"
-						more-search-results-label="{translate key="common.numberedMore"}"
-						:recipient-options="step.recipientOptions"
-						:recipients="step.recipients"
-						recipients-label="{translate key="semicolon" label=$recipientsLabel}"
-						remove-item-label="{translate key="common.removeItem"}"
-						searching-label="{translate key="common.searching"}"
-						search-results-label="{translate key="search.searchResults"}"
-						:separate-emails="step.anonymousRecipients"
-						:subject="step.subject"
-						subject-label="{translate key="semicolon" label=$subjectLabel}"
-						switch-to-label="{translate key="semicolon" label=$switchTolabel}"
-						switch-to-named-language-label="{translate key="common.switchToNamedItem"}"
-						:variables="step.variables"
-						@set="updateStep"
-					>
-
-						<template slot="description">
-							<h2>{{ step.name }}</h2>
-							<p v-html="step.description"></p>
-						</template>
-					</composer>
-				</template>
-				<template v-else-if="step.type === 'promoteFiles'">
-					<panel>
+						<panel-section v-else>
+							{capture assign="attachedFilesLabel"}{translate key="common.attachedFiles"}{/capture}
+							{capture assign="bccLabel"}{translate key="email.bcc"}{/capture}
+							{capture assign="ccLabel"}{translate key="email.cc"}{/capture}
+							{capture assign="subjectLabel"}{translate key="email.subject"}{/capture}
+							{capture assign="switchTolabel"}{translate key="common.switchTo"}{/capture}
+							{capture assign="recipientsLabel"}{translate key="email.to"}{/capture}
+							<composer
+								add-c-c-label="{translate key="common.addCCBCC"}"
```

Comment:
I think these `{translate}` tags need to be escaped, or a quote might break the template.